metrics/internal/collectors/cluster-advance-feature-use.go

```diff
@@ -88,9 +87,9 @@ func (s *StorageClassAdvancedFeatureProvider) AdvancedFeature(namespaces ...stri
 	return 0
 }
 
-func NewStorageClassAdvancedFeatureProvider(client *kubernetes.Clientset) AdvancedFeatureProvider {
+func NewStorageClassAdvancedFeatureProvider(opts *options.Options, client *kubernetes.Clientset) AdvancedFeatureProvider {
 	storageclassClient := client.StorageV1()
-	lw := cache.NewListWatchFromClient(storageclassClient.RESTClient(), "storageclasses", metav1.NamespaceAll, fields.Everything())
+	lw := cache.NewListWatchFromClient(storageclassClient.RESTClient(), "storageclasses", searchInNamespace(opts), fields.Everything())
```

Comment:
StorageClass is a cluster scoped resource. It does not have a namespace.

Reply:
Done